dengue_envs/envs/dengue_diagnostics.py

```diff
@@ -347,25 +342,34 @@ def get_daily_cases(self):
         Generate the daily cases based on an epidemic curve
         """
         for t in range(self.epilength):
-            if t == 0:
-                dcases_x = self.dengue_dist_x.rvs(int(self.dengue_curve[t]))
-                dcases_y = self.dengue_dist_y.rvs(int(self.dengue_curve[t]))
-                ccases_x = self.chik_dist_x.rvs(int(self.chik_curve[t]))
-                ccases_y = self.chik_dist_y.rvs(int(self.chik_curve[t]))
-            else:
-                dcases_x = self.dengue_dist_x.rvs(int(self.dengue_curve[t]-self.dengue_curve[t-1]))
-                dcases_y = self.dengue_dist_y.rvs(int(self.dengue_curve[t]-self.dengue_curve[t-1]))
-                ccases_x = self.chik_dist_x.rvs(int(self.chik_curve[t]-self.chik_curve[t-1]))
-                ccases_y = self.chik_dist_y.rvs(int(self.chik_curve[t]-self.chik_curve[t-1]))
+            # if t == 0:
+            #     dcases_x = self.dengue_dist_x.rvs(int(self.dengue_curve[t]))
+            #     dcases_y = self.dengue_dist_y.rvs(int(self.dengue_curve[t]))
+            #     ccases_x = self.chik_dist_x.rvs(int(self.chik_curve[t]))
+            #     ccases_y = self.chik_dist_y.rvs(int(self.chik_curve[t]))
+            # else:
+            #     dcases_x = self.dengue_dist_x.rvs(int(self.dengue_curve[t]-self.dengue_curve[t-1]))
+            #     dcases_y = self.dengue_dist_y.rvs(int(self.dengue_curve[t]-self.dengue_curve[t-1]))
+            #     ccases_x = self.chik_dist_x.rvs(int(self.chik_curve[t]-self.chik_curve[t-1]))
+            #     ccases_y = self.chik_dist_y.rvs(int(self.chik_curve[t]-self.chik_curve[t-1]))
+
+            dcases_x = self.dengue_dist_x.rvs(int(self.dengue_curve[t]))
```

Comment:
These cases are not ideal because they will decrease in number after the peak of the epidemic. My first solution is not completely correct either, because it doesn't take into account the cases that recover. I know how to fix this and can do this on another PR